### DISCUSSION TITLE
biubiu加速器今天怎么不加速日邦了？

### DISCUSSION THREAD

**Discussion by kallen1314** [share121/inter-knot#1126] [其他]
今天久违地想打开日邦打算看看，结果biubiu加速器没有加速的选项了，怎么会这样？
![85362FEAAF453A73D409A4FE07851FF3](https://github.com/user-attachments/assets/9b3c85e1-1371-4b44-b84f-bc25ccb25874)

**Comment by hzjwork** [comment]
![31e11c7ef2e332055672d59f2682207](https://github.com/user-attachments/assets/b06713fa-0a0c-4b76-ad66-f5e98c9d9c0f)
我这可以啊

**Reply by kallen1314** [reply]
解决了，在“找游戏”里重新加入了

**Comment by kallen1314** [comment]
区区20抽，我投了！！为了生日签
![CF4BCEE88FC3EA15C7E752B17D6D9A64](https://github.com/user-attachments/assets/fc2e2259-711b-41b8-8aae-514da2530c97)
![FC74D4D98985026DBF251EFB4843770E](https://github.com/user-attachments/assets/2a02a191-f1aa-4f24-83ca-81e0148e2639)

**Comment by hzjwork** [comment]
hhh刚看到，我就6000水我也抽了20抽把两个拿了，甚至连个四星都没出
